src/tests/unit/tests/popup/components/diagnostic-view-toggle.test.tsx

```diff
@@ -278,14 +287,14 @@ describe('DiagnosticViewToggleTest', () => {
 
             const component = React.createElement(DiagnosticViewToggle, props);
 
-            const testObject = TestUtils.renderIntoDocument(component);
+            const wrapper = render(component);
 
-            (testObject as any)._isMounted = true;
+            const toggle = wrapper.getByRole('switch');
+            expect(toggle).not.toHaveFocus();
 
-            const onFocusHandlerFunction = (testObject as any).onFocusHandler;
-            onFocusHandlerFunction();
+            fireEvent.focus(toggle);
 
-            expect((testObject as any).state.isFocused).toBeTruthy();
+            expect(toggle).toHaveFocus();
         });
 
         it('onBlurHandler', () => {
```

Comment:
I don't think either `onBlurHandler` or `onBlurToggleHandler` tests are testing the behavior we want to test. We know that the toggle won't have focus on initial render, but that isn't related to blur. And blurring the toggle after initial render doesn't change anything because it didn't have focus to begin with.